core/cont/inc/TVirtualCollectionProxy.h

```diff
@@ -74,6 +74,9 @@ class TVirtualCollectionProxy {
    virtual TVirtualCollectionProxy* Generate() const = 0; // Returns an object of the actual CollectionProxy class
    virtual ~TVirtualCollectionProxy() {};
 
+   // Reset the info gathered from StreamerInfos and value's TClass.
+   virtual Bool_t    Reset() { return kTRUE; };
```

Comment:
That the reset failing somehow ... I guess this could/should be replaced by throwing an exception but this code is not using them anywhere else ....